pkg/pillar/go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/anatol/smart.go v0.0.0-20241126061019-f03d79b340d2
 	github.com/andrewd-zededa/go-libzfs v0.0.0-20240304231806-6a64e99da97d
 	github.com/containerd/cgroups v1.1.0
```

Comment:
should we use `github.com/containerd/cgroups/v3/cgroup2/stats` here?

Reply:
@europaul I updated to v2 stats